lake-manifest.json

```diff
@@ -28,7 +28,7 @@
   {"git":
    {"url": "https://github.com/leanprover/std4",
     "subDir?": null,
-    "rev": "67855403d60daf181775fa1ec63b04e70bcc3921",
+    "rev":  "67855403d60daf181775fa1ec63b04e70bcc3921",
```

Comment:
Does lake / lean-cache hash the textual content of this file, as opposed to hashing the decoded json?

Reply:
it hashes the text